docs/content/guides/developer/sui-101/simulating-refs.mdx

```diff
@@ -1,6 +1,126 @@
 ---
 title: Simulating References
-draft: true
+description: Use the borrow module in the Sui framework to include objects by reference in your programmable transaction blocks.
 ---
 
-Content coming soon
\ No newline at end of file
+Everything on the Sui blockchain is an object. When you develop Move packages for the Sui network, you're typically manipulating or using on-chain objects in some way through functionality available in the Sui API. For most API functions, you provide an object     
+
+References are a key construct when programming in Move and on Sui. Most of the available Sui API methods take objects by reference, which means you pass a pointer to the object instead of the object itself. 
```

Comment:
> API methods

remove methods?

> which means you pass a pointer to the object instead of the object itself

if pointer makes sense to you that is fine, otherwise not sure it clarifies much